src/pages/blog.js

```
@@ -9,64 +9,56 @@ import '../assets/css/blog.css'
+            <div className="card">
+              {image && (
+                <GatsbyImage className="post-img" image={image} alt={title} />
+              )}
+              <div className="mainc">
+                <h3>
+                  <Link to={node.fields.slug}>{title}</Link>
+                </h3>
+                <small>{node.frontmatter.date}</small>
+                <div
+                  className="container"
+                  dangerouslySetInnerHTML={{ __html: shorten(node.html, 80) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)